src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -307,13 +312,14 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
       {!backgroundImage ? (
```

Comment:
can invert to avoid negation:

`! condition ? B : A`   -> ` condition ? A : B`

Reply:
fixed in a4aa1c4f4ec90d75594111a0ed9e49045dfe7cd6